source/reference/verification.txt

```diff
@@ -0,0 +1,211 @@
+
+####################
+Verify Data Transfer
+####################
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Following a sync, verify the successful transfer of your data before you switch
+your application from using the source cluster to the destination cluster. You
+can verify your data transfer using document counts, hash comparison, document
+comparison, or the Migration Verifier.
+
+Use Cases
+=========
+
+You should verify your data after every sync. This is important in
+cases where you plan to move your application load from the source to the
+destination cluster.
+
+Verification methods include the comparison of
+:ref:`document counts <c2c-verify-doc-counts>`,
+comparison of :ref:`database hashes <c2c-verify-hash-comp>`,
+scripting :ref:`document comparisons <c2c-verify-doc-comp>`,
+and the :ref:`Migration Verifier <c2c-verify-migration-verifier>`.
+The specific method you use to verify your data depends on your application
+workload and the complexity of the data.
+
+Details
+=======
+
+.. _c2c-verify-doc-counts:
+
+Document Counts
```

Comment:
I'm a little confused as to whether this is a task page or a concept page. Looking at our IA templates, the Use Cases and Details sections align with the [concept](https://www.mongodb.com/docs/meta/style-guide/information-types/prototypes/concept/) page, but we're also kind of showing users how to accomplish the task here.

If we're using Use Cases and Details sections and treating this as a conceptual page, I'd maybe expect to see the specific methods appear in a Task section, with each having their own page.

Reply:
Refactored to split the examples out into tasks.